modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -4,12 +4,18 @@
             [clojure.string :as str]
             [clojure.tools.logging :as log]
             [medley.core :as m]
+            [metabase.db.metadata-queries :as metadata-queries]
             [metabase.driver :as driver]
             [metabase.driver.bigquery-cloud-sdk.common :as bigquery.common]
             [metabase.driver.bigquery-cloud-sdk.params :as bigquery.params]
             [metabase.driver.bigquery-cloud-sdk.query-processor :as bigquery.qp]
             [metabase.driver.sync :as driver.s]
             [metabase.models :refer [Database Table] :rename {Table MetabaseTable}] ; Table clashes with the class below
+            [metabase.driver.sql-jdbc.sync.describe-database :as describe-database]
+            [metabase.driver.sync.fingerprint :as sync.f]
+            [metabase.models :refer [Database]]
```

Comment:
there's already a `metabase.models :refer {Database Table}` above

Reply:
Yeah this was from a botched push.  It should be fixed now.